packages/perpswap/src/contracts/vault.rs

```diff
@@ -20,7 +20,7 @@ pub struct InstantiateMsg {
 #[derive(serde::Serialize, serde::Deserialize, Debug, Clone)]
 #[serde(rename_all = "snake_case")]
 pub struct Config {
-    /// Denomination of the USDC token (e.g., "usdc")
+    /// Denomination of the USDC token. Varies by chain (e.g., "uusdc" on Osmosis, "usdc" on Noble, ...)
```

Comment:
Are you sure that's the denom on Osmosis?

<img width="419" alt="image" src="https://github.com/user-attachments/assets/dd97f333-0036-484c-afa7-d47347be69ad" />

https://www.mintscan.io/osmosis/assets/